### ISSUE TITLE
[screen reader] Pagination on search results and on the record page

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#4898]
### Steps to recreate
- Turn on Voice over
- Do a search https://catalog.princeton.edu/catalog?search_field=all_fields&q=math
- Find the pagination button 'Previous', 'Next'. Voice over reads Previous next on the search results page and on the record page.

### Issues
- Visually impaired users find that the pagination Previous Next should be more precise and read:
     - On the search results page 'Previous results' , 'Next results'. 
     - On the record page 'Previous record', 'Next record'

### WCAG Conformance level (A, AA, AAA)
N/A

### User impact
- Visually impaired users find that the pagination Previous Next should be more precise and read:
     - On the search results page 'Previous results' , 'Next results'. 
     - On the record page 'Previous record', 'Next record'

### Recommended fix
Rename aria on search results and record page to much:
     - On the search results page 'Previous results' , 'Next results'. 
     - On the record page 'Previous record', 'Next record' 

### Acceptance criteria

      

- [ ] On the search results page the screen reader should read 'Previous results' , 'Next results'. 
- [ ] On the record page the screen reader should read 'Previous record', 'Next record' 
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave

### Notes
Related to https://github.com/pulibrary/orangelight/issues/4888
